utils/merge_and_push_translations.sh

```diff
@@ -54,89 +54,141 @@ fi
 
 echo "Working tree is clean, continuing"
 
-# Print all commands.
-set -x
+update_authors() {
+  # Update authors file
+  if python3 utils/update_authors.py; then
+    echo "Updated authors"
+  else
+    echo "Failed updating authors"
+    exit 1
+  fi
+  # Fix formatting for Lua
+  python3 utils/fix_formatting.py --lua --dir data/i18n
+  python3 utils/fix_formatting.py --lua --dir data/txts
+}
 
-# Update source catalogs
-python3 utils/buildcat.py
+update_appdata() {
+  # Update appdata
+  if python3 utils/update_appdata.py; then
+    echo "Updated appdata"
+  else
+    echo "Failed updating appdata"
+    exit 1
+  fi
+}
+
+update_statistics() {
```

Comment:
This would probably work. At least I can find no real corner case.
(If somebody edits a po file outside transifex, it is overwritten by transifex pull.)
(If somebody edits update_translation_statistics.py, the changes to the statistics file are hopefully committed as well. If not they will come on a later 

I want to make sure all is in sync after the script has run. But here it can not really get out of sync.

Reply:
Well, if we want to be sure, then it's best to always run these updates, in both turns. :)